manifests/00-crd-extension-console-notification-banner.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: consolenotificationbanners.console.openshift.io
+  annotations:
+    displayName: ConsoleNotification
+    description: Extension for configuring openshift web console notifications.
+spec:
+  group: console.openshift.io
+  version: v1
+  versions:
+  - name: v1
+    served: true
+    storage: true
+  scope: Cluster
+  names:
+    plural: consolenotifications
+    singular: consolenotification
+    kind: ConsoleNotification
+    listKind: ConsoleNotificationList
+  additionalPrinterColumns:
+  - name: Text
+    type: string
+    JSONPath: .spec.text
+  - name: Age
+    type: date
+    JSONPath: .metadata.creationTimestamp
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
+          description: Represents console notification spec
+          required:
+          - text
+          - position
+          properties:
+            text:
+              type: string
+              description: Text of the notification
+            position:
+              type: string
+              description: The location of the notification (top, bottom, topBottom)
+              pattern: '\btop\b|\bbottom\b|\btopBottom\b'
+            link:
+              type: object
+              description: Object that holds notification link details
+              required:
+              - href
```

Comment:
`- text` ought to be required as well?  can't easily have a link w/o text.

Reply:
There is a default text (More info) in the console.

Reply:
Ok, trying to decide if we need to doc that on the api object or not.